src/index.html

```diff
@@ -3,11 +3,311 @@
+                    alt="Woman holding Bose speaker"
+                  >
+                  <img
+                    class="browse-bose__image browse-bose__image--col-3-4 browse-bose__image--col320-2-3"
+                    src="images/photos/4.jpg"
+                    alt="Blue Bose speaker"
+                  >
+                </div>
+                <h3 class="browse-bose__department">
+                  Speakers
+                </h3>
+              </a>
+            </div>
+
+            <div class="browse-bose__card">
+              <a class="browse-bose__link" href="#">
+                <div class="browse-bose__collection">
+                  <img
+                    class="browse-bose__image browse-bose__image--col-1-2 browse-bose__image--col320-1-2"
+                    src="images/photos/5.jpg"
+                    alt="Man wearing Bose audio sunglasses"
+                  >
+                  <img
+                    class="browse-bose__image browse-bose__image--col-2-4 browse-bose__image--col320-2-3"
+                    src="images/photos/6.jpg"
+                    alt="Woman wearing Bose audio sunglasses"
+                  >
+                </div>
+                <h3 class="browse-bose__department">
+                  Audio sunglasses
+                </h3>
+              </a>
+            </div>
+          </div>
+
+          <div class="browse-bose__button-wrapper">
+            <a href="#" class="browse-bose__button">
+              View all
+            </a>
+          </div>
+        </div>
+      </section>
+
+      <section class="promo" id="testimonials">
+        <div class="promo__wrapper">
+          <h2 class="promo__title title">
+            Why buy direct from Bose
+          </h2>
+
+          <div class="promo__content">
+            <div class="promo_list">
+              <p class="promo__points">
+                Free 2-day shipping and returns
+              </p>
+              <p class="promo__points">
+                90-day risk-free trial
+              </p>
+              <p class="promo__points">
+                World class customer service
+              </p>
+              <p class="promo__points">
+                My Bose account management
+              </p>
+            </div>
+            <div class="promo__text">
+              <p>
+                A great product is more than what’s in the box. It’s a promise of premium performance, world-class support, and everything you expect from a trusted brand. It’s just one of many reasons why you’ll shop with confidence on Bose.com.
+              </p>
+            </div>
+          </div>
+
+        </div>
+      </section>
+
+      <section class="contact-us" id="contacts">
+        <div class="contact-us__wrapper">
+          <h2 class="contact-us__title title">
+            Contact us
+          </h2>
+
+          <div class="contact-us__content-wrapper">
+            <div class="contact-us__submit">
+              <form
```

Comment:
>15	Page shouldn't be reloaded on form submit
(`action="#" method="get"` OR https://developer.mozilla.org/en-US/docs/Web/API/Event/preventDefault)

Reply:
Done.

Reply:
@rHashTag your form still reloads the page
use .js to make sure it's not

```
const form = document.querySelector('<form selector>');

form.addEventListener('submit', function(event) {
  event.preventDefault();
  form.reset();
});
```

Reply:
Done!

Also updated the 100vh for mobile and tablet views, just to be consistent.